src/test/java/service/impl/ReadDataServiceImplTest.java

```diff
@@ -0,0 +1,38 @@
+package service.impl;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.ReadDataService;
+
+public class ReadDataServiceImplTest {
+    private static final String EMPTY_FILE_PATH = "src/test/java/resources/emptyFile_test.csv";
+    private static final String VALID_PATH = "src/test/java/resources/correctFile_test.csv";
```

Comment:
```suggestion
    private static final String EMPTY_FILE_PATH = "src/test/resources/emptyFile_test.csv";
    private static final String VALID_PATH = "src/test/resources/correctFile_test.csv";
```
Your test data should be placed under 'src/test/resources' directory